From e82ef8ec38e298998d8bff59973a35412c1fca98 Mon Sep 17 00:00:00 2001
From: George Florea Banus <georgefb899@gmail.com>
Date: Mon, 27 Dec 2021 19:01:45 +0200
Subject: [PATCH] remove mpv_opengl_init_params.extra_exts field

---
 src/mpvobject.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mpvobject.cpp b/src/mpvobject.cpp
index cee6ee7..d20d73e 100644
--- a/src/mpvobject.cpp
+++ b/src/mpvobject.cpp
@@ -79,7 +79,7 @@ QOpenGLFramebufferObject * MpvRenderer::createFramebufferObject(const QSize &siz
     // init mpv_gl:
     if (!obj->mpv_gl)
     {
-        mpv_opengl_init_params gl_init_params{get_proc_address_mpv, nullptr, nullptr};
+        mpv_opengl_init_params gl_init_params{get_proc_address_mpv, nullptr};
         mpv_render_param params[]{
             {MPV_RENDER_PARAM_API_TYPE, const_cast<char *>(MPV_RENDER_API_TYPE_OPENGL)},
             {MPV_RENDER_PARAM_OPENGL_INIT_PARAMS, &gl_init_params},
-- 
2.30.2.windows.1

